index.js

```diff
@@ -4,24 +4,21 @@ var https = require('https')
 var fs = require('fs')
 
 function getImageData (json) {
-  var images = json.images || json.album.images
-  var name = json.images ? json.name : json.album.name
-
-  var image = images.reduce((x, y) => {
-    return x.height * x.width > y.height * y.width ? x : y
-  })
-
-  name = `${name}-${image.width}x${image.height}`
+  let image = {
+    name: `${json.title}-${json.thumbnail_width}x${json.thumbnail_height}`,
+    url: json.thumbnail_url
+  }
 
-  return { name: name, url: image.url }
+  return image;
```

Comment:
Nice! However, now that we're not doing any magic in the function, would you mind rewriting it as an [arrow function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)? 😇